app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -303,6 +310,19 @@ public void onCreate(Bundle savedInstanceState) {
         new initAllPhotos().execute();
         displayData(getIntent().getExtras());
         checkNothing();
+
+        navigationView.setOnNavigationItemSelectedListener(new BottomNavigationView.OnNavigationItemSelectedListener() {
+            @Override
+            public boolean onNavigationItemSelected(@NonNull MenuItem item) {
+                int itemID = item.getItemId();
+                if(itemID==R.id.navigation_home){
+                    startActivity(new Intent(LFMainActivity.this, LFMainActivity.class));
```

Comment:
Don't restart the activity on pressing the home button, just switch to Albums mode if it is in photos mode.

Reply:
@anantprsd5 Sorry to interrupt, but can you explain why switching to Album Mode, is considered to be a good practice? Sorry for being lame.

Reply:
@anantprsd5 is that what you want?
Please check this gist :)
https://gist.github.com/aditya98ak/8553e9279f17387aff2e518802be73ea

Reply:
Why would you want to recreate the same activity when you have an option to manage the contents in that activity. For the hidden folders and the all photos mode and for the photos mode, simply put an if statement and switch to albums mode. Recreating the activity will take much more time rather than just switching to albums mode.

Reply:
@anantprsd5 I have updated the gist  - https://gist.github.com/aditya98ak/8553e9279f17387aff2e518802be73ea

Everything is working fine now.(I am not recreating activity now)
If this is fine, I'll commit the changes asap :)

Reply:
Sure. Please commit the changes. Thanks!